governance/README.md

```diff
@@ -0,0 +1,7 @@
+# Governance
+Environmental Sustainability Working Group is a CNCF Working Group and reports to the Technical Oversight Comittee (TOC).
+
+* [Charter](../charter.md) - mission and scope
+* [Roles](roles.md) - the work of the group is facilitated by Chairs, Technical Leads, and active group members
```

Comment:
Sounds great.  Oh, do we want another one for "Media Lead" or something particular for handling conferences and keeping us organized?

Reply:
How about if the media lead is a maintainer (role maintainer)? Alsom, It might be useful to define further maintainer "specifics", e.g. for blog posts / articles, Github governance, general collaboration

Reply:
From the 11/2 meeting: suggest a Tech Lead for Outreach, TAGs do not have maintainers :)